<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LinkwPage</class>
 <widget class="QWidget" name="LinkwPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>835</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>997</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>997</width>
    <height>835</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame_6">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>180</y>
     <width>300</width>
     <height>260</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>260</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>260</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_8">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>180</y>
     <width>400</width>
     <height>260</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>260</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>260</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_9">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>440</y>
     <width>300</width>
     <height>260</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>260</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>260</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_7">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>9</y>
     <width>400</width>
     <height>170</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>170</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>170</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_14">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>180</y>
     <width>250</width>
     <height>260</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>260</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>250</width>
     <height>260</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_5">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>10</y>
     <width>300</width>
     <height>170</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>170</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>170</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_10">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>440</y>
     <width>400</width>
     <height>260</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>260</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>260</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_16">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>440</y>
     <width>250</width>
     <height>260</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>260</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>250</width>
     <height>260</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_13">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>9</y>
     <width>140</width>
     <height>170</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>170</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>140</width>
     <height>170</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>720</y>
     <width>991</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalBanner_2">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetFixedSize</enum>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
   </layout>
  </widget>
  <widget class="QFrame" name="frame_2coininfo">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>13</y>
     <width>128</width>
     <height>74</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>128</width>
     <height>74</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>128</width>
     <height>74</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
